### PR TITLE
refactor: Prepare the project to use custom icons (#40)

### LINKED ISSUE #40: Prepare the project to use our custom icons
As we have our own icons already in the repo we can remove @chakra-ui/icons package we used before.

**Acceptance criteria:**
- There is no @chakra-ui/icons package in the repository.
- All icons in stories are imported from the `ui/src/icons` directory.
- There is a new `generate` command in the main `package.json` file with content `yarn workspaces foreach -vpi run generate`.
- `yarn bootstrap` is the `heroku-postbuild` script in `package.json` is replaced by `yarn generate`.
- `yarn generate` is called after `yarn install` in all GitHub workflows.

### PR DESCRIPTION
**Pull request by hoolek77**
Closes #40

### CODE REVIEW

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -11,29 +11,35 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: install dependencies 
+      - name: install dependencies
         run: yarn install --immutable
-      - name: run affected tests 
+      - name: generate workspaces
```
Comment (KonradSzwarc) [inline .github/workflows/workflow.yml:16]: As this command doesn't generate new workspaces I think "create autogenerated content" could be a better name

File: .github/workflows/workflow.yml [hunk 2]
```diff
@@ -11,29 +11,35 @@ jobs:
...
+      - uses: actions/checkout@v2
+      - name: install node v14
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - name: install dependencies
+        run: yarn install --immutable
+      - name: create autogenerated content
+        run: yarn generate
+      - name: tsc compile webside
```
Comment (mecies) [inline .github/workflows/workflow.yml:44]: You probably meant `website` 😄

**Review by mecies — APPROVED** [review]
LGTM, please correct the typo ✅ 
I recommend **cSpell** `streetsidesoftware.code-spell-checker` extension to prevent such situations.

**Review by mecies — APPROVED** [review]
Looks good to me ✅ 
Please keep in mind that I'm not fully onboarded and I do not know project dependencies etc, code looks good 😄

**Review by KonradSzwarc — APPROVED** [review]

**Review by KonradSzwarc — CHANGES_REQUESTED** [review]
Something with Heroku is broken. I will investigate it

**Comment by KonradSzwarc** [comment]
Ok, I got the problem
CleanShot 2021-07-09 at 22 17 40@2x

**Review by KonradSzwarc — APPROVED** [review]

**Review by lagdush — APPROVED** [review]

**Pull request merged**